cookbooks/add_memory_using_qdrant_cloud.py

```diff
@@ -0,0 +1,41 @@
+# This example shows how to use vector config to use QDRANT CLOUD
+import os
+from dotenv import load_dotenv
+from mem0 import Memory
+
+# Loading my API_KEY for OPENAI
```

Comment:
Here you can write like `Loading OpenAI API Key`

Reply:
done

Reply:
Hi Dev, if you are suggesting to remove the below lines, I think it will be confusing for the developers. I suggest to keep them, but would go with what you decide. I thought this was the open review comment you referred below

load_dotenv()
OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')